builtin/providers/azurerm/resource_arm_network_interface_card.go

```diff
@@ -53,6 +53,12 @@ func resourceArmNetworkInterface() *schema.Resource {
 				Computed: true,
 			},
 
+			"private_ip_address": &schema.Schema{
```

Comment:
We are only using this part of the schema in the Read func. Is this not allowed to be set specifically? If not, then we can just make it Computed, and remove optional